cmd/examples/server.cpp

```diff
@@ -56,26 +56,28 @@ namespace qserver_vars {
      * Map of subscribes set by namespace and track name hash
      *      Set<subscribe_who> = subscribe_active[track_namespace_hash][track_name_hash]
      */
-    struct SubscribeWho
+    struct SubscribeInfo
     {
         uint64_t connection_handle;
         uint64_t subscribe_id;
         uint64_t track_alias;
 
-        bool operator<(const SubscribeWho& other) const
+        bool operator<(const SubscribeInfo& other) const
         {
-            return connection_handle < other.connection_handle && subscribe_id << other.subscribe_id;
+            return connection_handle < other.connection_handle ||
```

Comment:
@shenning00  found this out where we weren't removing state. crazy that this wasn't caught sooner.